roles/openshift_control_plane/tasks/main.yml

```diff
@@ -129,6 +129,12 @@
   # copy to this location to bypass initial bootstrap request
   - /etc/origin/node/node.kubeconfig
 
+- name: Pre-pull Origin image
+  command: "atomic pull docker:{{ osm_image }}"
```

Comment:
Is this going to work with crio-only installs?  IDK.

Reply:
It won't, correct. I don't CRI-O-only installs are even working though